packages/frontend-2/pages/dashboard.vue

```diff
@@ -0,0 +1,129 @@
+          :key="`quick-start-${index}`"
+          :title="quickStartItem.title"
+          :description="quickStartItem.description"
+          :buttons="quickStartItem.buttons"
+        />
+      </div>
+    </section>
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">Recently updated projects</h2>
+      <div class="flex flex-col md:grid md:grid-cols-3 gap-3 pt-5">
+        <DashboardProjectCard
+          v-for="project in projects"
+          :key="project.id"
+          :project="project"
+        />
+      </div>
+    </section>
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">News &amp; tutorials</h2>
+      <div
+        class="flex flex-col gap-y-3 md:grid md:grid-cols-2 lg:grid-cols-4 gap-3 pt-5"
+      >
+        <DashboardTutorialCard
+          v-for="tutorial in tutorials"
+          :key="tutorial.id"
+          :tutorial="tutorial"
+        />
+      </div>
+    </section>
+  </div>
+</template>
+<script setup lang="ts">
+import { dashboardProjectsQuery } from '~~/lib/dashboard/graphql/queries'
+import type { TutorialCard, QuickStartItem } from '~~/lib/dashboard/helpers/types'
+import { useQuery } from '@vue/apollo-composable'
+import { useMixpanel } from '~~/lib/core/composables/mp'
+import GhostContentAPI from '@tryghost/content-api'
+import { docsPageUrl, forumPageUrl } from '~~/lib/common/helpers/route'
+import type { ManagerExtension } from '~~/lib/common/utils/downloadManager'
+import { downloadManager } from '~~/lib/common/utils/downloadManager'
+
+useHead({ title: 'Dashboard' })
+
+const config = useRuntimeConfig()
+const mixpanel = useMixpanel()
+const { result: projectsResult } = useQuery(dashboardProjectsQuery)
+
+const ghostContentApi = new GhostContentAPI({
+  url: 'https://speckle.systems',
+  key: config.public.ghostApiKey,
+  version: 'v5.0'
+})
+
+const tutorials = ref<TutorialCard[]>([])
+const quickStartItems = shallowRef<QuickStartItem[]>([
+  {
+    title: 'Install Speckle manager',
+    description: 'Use our Manager to install and manage Connectors with ease.',
+    buttons: [
+      {
+        text: 'Download for Windows',
+        onClick: () => onDownloadManager('exe')
+      },
+      {
+        text: 'Download for Mac',
+        onClick: () => onDownloadManager('dmg')
+      }
+    ]
+  },
+  {
+    title: "Don't know where to start?",
+    description: "We'll walk you through some of most common usage scenarios.",
+    buttons: [
+      {
+        text: 'Open documentation',
+        props: { to: docsPageUrl, external: true }
+      }
+    ]
+  },
+  {
+    title: 'Have a question you need answered?',
+    description: 'Submit your question on the forum and get help from the community.',
+    buttons: [
+      {
+        text: 'Ask a question',
+        props: { to: forumPageUrl, external: true }
+      }
+    ]
+  }
+])
+
+const projects = computed(() => projectsResult.value?.activeUser?.projects.items)
+
+async function fetchTutorials() {
```

Comment:
I was thinking to do this server-side, but it's non-block non-required content so I changed to being fetched async. Probably we can have some nicer loader for this later